web/views/_footer.erb

```diff
@@ -15,7 +15,18 @@
             <p class="navbar-text"><a rel=help href="https://github.com/sidekiq/sidekiq/wiki">docs</a></p>
           </li>
           <li>
-            <p class="navbar-text"><a rel=external href="https://github.com/sidekiq/sidekiq/tree/main/web/locales"><%= locale %></a></p>
+            <form id="locale-form" action="<%= root_path %>change_locale" method="post">
+              <%= csrf_tag %>
+              <select id="locale-select" class="locale-select" name="locale">
+                <% available_locales.each do |locale_option| %>
+                  <% if locale_option === locale %>
```

Comment:
Be careful. By using triple ===, I believe you may be risking a security hole, allowing anyone to inject any value for the `locale` parameter in the session. Thankfully you aren't rendering it.